components/automate-deployment/pkg/backup/runner.go

```diff
@@ -892,6 +893,30 @@ func (r *Runner) startBackupOperations(ctx context.Context) {
 	ctx, cancel := context.WithDeadline(context.Background(), deadline)
 	defer cancel()
 
+	if r.converger != nil {
+		defer func() {
```

Comment:
[nit] given how long this function already is, maybe pull this out into a function